src/main/java/org/jenkinsci/plugins/gitclient/GitUsernamePasswordBinding.java

```diff
@@ -43,14 +43,18 @@ public GitUsernamePasswordBinding(String credentialsId) {
         return StandardUsernamePasswordCredentials.class;
     }
 
+    private static void setUnixNodeType(boolean value) {
+        unixNodeType = value;
```

Comment:
@MarkEWaite, I am a bit doubtful regarding the use of `unixNodeType` as a class variable and not as instance variable. The scenario which I think could cause a problem is, when the user adds multiple `GitUsernamePassword` bindings and executes these in different operating system environments, won't  `unixNodeType` value be same for the all the `GitUsernamePassword` bindings since it's a class variable or am I lacking some insight over the concept of Descriptors and Describables?

I thought of making `unixNodeType` as an instance type variable and passing it to the `GenerateGitScript` constructor as a parameter.

Reply:
I am also doubtful of using it as a class variable rather than an instance variable.  The code change that I submitted was only done so that I could get an incremental build from ci.jenkins.io in order to do some exploratory tests on my Jenkins instance.